test/parallel/test-bootstrap-modules.js

```diff
@@ -42,9 +45,11 @@ const expectedModules = new Set([
   'NativeModule internal/constants',
   'NativeModule internal/encoding',
   'NativeModule internal/errors',
+  'NativeModule internal/event_target',
   'NativeModule internal/fixed_queue',
   'NativeModule internal/fs/dir',
   'NativeModule internal/fs/utils',
+  'NativeModule internal/histogram',
```

Comment:
Please lazy load all modules that are not essential. Otherwise we'll increase our startup time.

Reply:
See https://github.com/nodejs/node/pull/33527#discussion_r434932802